gnark/gnark-jni/gnark-eip-2537.go

```diff
@@ -339,93 +247,49 @@ func eip2537blsG2Add(javaInputBuf, javaOutputBuf, javaErrorBuf *C.char, cInputLe
 	inputLen := int(cInputLen)
 	errorLen := int(cOutputLen)
```

Comment:
we should fix these while we are in here, re cOutputLen vs cErrorLen.  They are both called with 256, but safer/more correct to use the actual cErrorLen